components/clarinet-format/tests/golden-intended/alex-transfer.clar

```diff
@@ -0,0 +1,134 @@
+;; https://github.com/alexgo-io/alex-v1/blob/dev/clarity/contracts/stx404-token/token-stx404.clar#L60-L94
+(define-public (transfer
+        (amount-or-id uint)
+        (sender principal)
+        (recipient principal)
+    )
+    (begin
+        (asserts! (is-eq sender tx-sender) err-not-authorised)
+        (if (<= amount-or-id max-supply) ;; id transfer
+            (let (
+                (check-id
+                    (asserts! (is-id-owned-by-or-default amount-or-id sender)
+                        err-invalid-id
+                    )
+                )
+                (owned-by-sender (get-owned-or-default sender))
+                (owned-by-recipient (get-owned-or-default recipient))
+                (id-idx (unwrap-panic (index-of? owned-by-sender amount-or-id)))
+            )
+                (map-set owned sender (pop owned-by-sender id-idx))
+                (map-set owned recipient
+                    (unwrap-panic
+                        (as-max-len? (append owned-by-recipient amount-or-id)
+                            u10000
+                        )
+                    )
+                )
+                (try! (ft-transfer? stx404 one-8 sender recipient))
+                (try! (nft-transfer? stx404nft amount-or-id sender recipient))
+                (ok true)
+            )
+            (let (
+                (balance-sender (unwrap-panic (get-balance sender)))
+                (balance-recipient (unwrap-panic (get-balance recipient)))
+                (check-balance
+                    (try! (ft-transfer? stx404 amount-or-id sender recipient))
+                )
+                (no-to-treasury
+                    (- (/ balance-sender one-8)
+                        (/ (- balance-sender amount-or-id) one-8)
+                    )
+                )
+                (no-to-recipient
+                    (- (/ (+ balance-recipient amount-or-id) one-8)
+                        (/ balance-recipient one-8)
+                    )
+                )
+                (owned-by-sender (get-owned-or-default sender))
+                (owned-by-recipient (get-owned-or-default recipient))
+                (ids-to-treasury
+                    (if (is-eq no-to-treasury u0)
+                            (list)
+                            (unwrap-panic
+                            (slice? owned-by-sender
+                                (- (len owned-by-sender) no-to-treasury)
+                                (len owned-by-sender)
+                            )
+                        )
+                        )
+                )
```

Comment:
There are some indentation issues here right? (double indent within `if` and mis-aligned closing parenthesis lines 58 and 59
```
               (ids-to-treasury
                    (if (is-eq no-to-treasury u0)
                        (list)
                        (unwrap-panic
                            (slice? 
                                owned-by-sender
                                (- (len owned-by-sender) no-to-treasury)
                                (len owned-by-sender)
                            )
                        )
                    )
                )
```

Reply:
The double indent _is_ intended. The tricky part here is that the double indent is for situations where the `if` is nested inside something, like

```
(key (if (...)
    (list) ;; double indent here
    ...
```

but since the line was too long, it got broken up onto separate lines..

Open to whatever changes, but I'll fix the `unwrap-panic` indenting since that's clearly wrong

Reply:
We handled this on a call. the double indent is gone now. resolving this issue